python/private/python_repository.bzl

```diff
@@ -15,7 +15,7 @@
 """This file contains repository rules and macros to support toolchain registration.
 """
 
-load("//python:versions.bzl", "FREETHREADED", "INSTALL_ONLY", "PLATFORMS")
+load("//python:versions.bzl", "FREETHREADED", "INSTALL_ONLY")
```

Comment:
It is so nice to see that gone, but Workspace may still need it?

Reply:
I don't follow -- the only usage of PLATFORMS in this file was on the attribute values. It isn't otherwise used.